scipy/stats/_continuous_distns.py

```diff
@@ -757,37 +757,74 @@ class burr_gen(rv_continuous):
 
     .. math::
 
-        f(x, c, d) = c d x^{-c-1} (1+x^{-c})^{-d-1}
+        f(x, c, d) = c d x^{-c - 1} / (1 + x^{-c})^{d - 1}
```

Comment:
Should this be `... (1 + x^{-c})^{d + 1}`?  That would match the implementation.